<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraderDialog</class>
 <widget class="QDialog" name="TraderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1198</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1198</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Województwo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="voivodeshipComboBox">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Powiat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="countyComboBox">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Miejscowosc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cityComboBox">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="infoTableView">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>821</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Imię i nazwisko</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Godziny pracy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(27, 27, 27);
border: none;</string>
        </property>
        <property name="text">
         <string>Telefon</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="workHoursLineEdit">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailLineEdit">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneLineEdit">
        <property name="styleSheet">
         <string notr="true">border: 5px solid white;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cleanPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(212, 212, 212);
}


QPushButton:pressed {
	
	background-color: rgb(167, 167, 167);
}</string>
   </property>
   <property name="text">
    <string>Wyczysc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(212, 212, 212);
}


QPushButton:pressed {
	
	background-color: rgb(167, 167, 167);
}</string>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(212, 212, 212);
}


QPushButton:pressed {
	
	background-color: rgb(167, 167, 167);
}</string>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyPushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(212, 212, 212);
}


QPushButton:pressed {
	
	background-color: rgb(167, 167, 167);
}</string>
   </property>
   <property name="text">
    <string>Modyfikuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPushButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>360</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(212, 212, 212);
}


QPushButton:pressed {
	
	background-color: rgb(167, 167, 167);
}</string>
   </property>
   <property name="text">
    <string>Znajdź</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
